src/js/view/floating/floating-controller.ts

```diff
@@ -239,6 +241,10 @@ export default class FloatingController {
         // Perform resize and trigger "float" event responsively to prevent layout thrashing
         this._model.trigger('forceResponsiveListener', {});
```

Comment:
`this._model.trigger('forceResponsiveListener', {});` is how we make the responsive listener run. It needs to run after you toggle classes.